src/app/lib/config/services/tigrinya.js

```diff
@@ -11,20 +11,20 @@ const service = {
   default: {
     lang: `ti`,
     articleAuthor: `https://www.facebook.com/bbcnewstigrinya`,
-    articleTimestampPrefix: 'Updated',
+    articleTimestampPrefix: 'ዝተመሓየሸ',
     atiAnalyticsAppName: 'news-tigrinya',
     atiAnalyticsProducerId: '91',
     brandName: 'BBC News ትግርኛ',
     product: 'BBC News ትግርኛ',
     defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png',
     defaultImageAltText: 'BBC News ትግርኛ',
     dir: `ltr`,
-    externalLinkText: ', external',
-    imageCaptionOffscreenText: 'Image caption, ',
-    videoCaptionOffscreenText: 'Video caption, ',
-    audioCaptionOffscreenText: 'Audio caption',
-    defaultCaptionOffscreenText: 'Caption, ',
-    imageCopyrightOffscreenText: 'Image source, ',
+    externalLinkText: ', ደጋዊ',
```

Comment:
I'm not sure if these should have a comma. In the spreadsheet there isn't one (and the others end with a colon instead)